### ISSUE TITLE
to sync the entire dropbox folder would you run it like this?

### ISSUE THREAD

**Issue by vesper8** [ldx/DBdownload#10]
I want to use this to update an old and very much out-of-sync folder but that also has a lot of good files that don't need replacing

last time i tried to just run the regular dropbox sync on it, it was disastrous as it started messing with the up-to-date cloud version and it was just a huge mess because of how big the task was

i'm hoping your solution will work better and then i can connect it to dropbox sync once i have an identical clone without having to redownload everything that doesn't need to be downloaded

tl;dr is this how i would run it?

`dbdownload -s / -t /users/MyUser/Dropbox`

**Comment by ldx** [comment]
I'd suggest you back up the destination folder first. Otherwise, yes, the command you provided should work.
